ONSdigital/dp-kafka
# Feature/init

### What

- Imported the go-ns/kafka folder from master branch, keeping the history.
  - In order to keep the git history, I followed this guide:
https://medium.com/@ayushya/move-directory-from-one-repository-to-another-preserving-git-history-d210fa049d4b 
  - But creating repo-B (this repo) using our dp-utils script

- Converted to go module with dependencies: 
  - go mod init
  - go build ./...
  - go test ./...

### How to review

- Check that the kafka folder in go-ns was correctly copied to the root of this repository.
- Check that you can see the older git history coming from go-ns.
- Run go build ./... and go test ./... -> should not change go.mod or go.sum

### Who can review

Anyone.
